src/posix/platform/radio_spinel.hpp

```diff
@@ -197,7 +197,29 @@ class RadioSpinel : public HdlcInterface::Callbacks
      */
     int8_t GetReceiveSensitivity(void) const { return mRxSensitivity; }
 
-#if OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_METRICS_ENABLE
+#if OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE
+    /**
+     * Enable the radio coex.
+     *
+     * @param[in] aInstance  The OpenThread instance structure.
+     * @param[in] aEnabled   TRUE to enable the radio coex, FALSE otherwise.
+     *
+     * @retval OT_ERROR_NONE     Successfully enabled.
+     * @retval OT_ERROR_FAILED   The radio coex could not be enabled.
+     *
+     */
+    otError SetCoexEnabled(bool aEnabled);
+
+    /**
+     * Check whether radio coex is enabled or not.
+     *
+     * @param[in] aInstance  The OpenThread instance structure.
+     *
+     * @returns TRUE if the radio coex is enabled, FALSE otherwise.
+     *
+     */
+    bool IsCoexEnabled(void);
```

Comment:
Declare as `const`.

Reply:
Function `IsCoexEnabled()` calls the function `Get()` to get the state of Coex. The function `Get()` is not a const function. So I can't declare it as `const`.